early-attack-detector-py/src/findings.py

```diff
@@ -23,7 +24,9 @@ def __init__(
             "oko_contract_explorer"
         ] = f"https://oko.palkeo.com/{contract_address}/"
         self.metadata["model_score"] = str(model_score)
-        self.metadata["model_threshold"] = str(model_threshold)
+        self.metadata["model_threshold"] = str(model_threshold),
```

Comment:
```suggestion
        self.metadata["model_threshold"] = str(model_threshold)
```
Comma needs to be removed, because it converts the metadata into a tuple